DataConnectors/Netskope/AzureFunctionNetskope/run.ps1

```diff
@@ -0,0 +1,198 @@
+<#  
+    Title:          Netskope Connector
+    Language:       PowerShell
+    Version:        1.0
+    Author(s):      Microsoft
+    Last Modified:  11/13/2020
+    Comment:        Initial Release
+
+    DESCRIPTION
+    This Function App calls the Netskope Platform API (https://innovatechcloud.goskope.com/docs/Netskope_Help/en/rest-api-v2-overview.html) to pull alert and events data. The response from the Okta API is recieved in JSON format. This function will build the signature and authorization header 
+    needed to post the data to the Log Analytics workspace via the HTTP Data Connector API. The Function App will post to the Netskope_CL table in the Log Analytics workspace.
+#>
+
+# Input bindings are passed in via param block.
+param($Timer)
+
+# Get the current universal time in the default string format.
+$currentUTCtime = (Get-Date).ToUniversalTime()
+
+# The 'IsPastDue' property is 'true' when the current function invocation is later than scheduled.
+if ($Timer.IsPastDue) {
+    Write-Host "PowerShell timer is running late!"
+}
+
+# Function to contruct the Netskope Uri for alerts, event types, and to accomodate for pagination
+function GetUrl ($uri, $ApiKey, $StartTime, $EndTime, $LogType, $Page){
```

Comment:
$skip needs to be passed in as a parameter as well?

Reply:
this isn't necessary, but I have added for code comprehensibility